modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/webContentsGenerator.js

```diff
@@ -4,44 +4,61 @@ import {
 	WEB_CONTENTS,
 	WEB_CONTENTS_BY_LIFERAY_VERSION,
 } from './constants';
-const getDXPVersionWebContent = (dxpVersion) =>
-	WEB_CONTENTS_BY_LIFERAY_VERSION[dxpVersion];
+
+const getDXPVersionWebContent = (dxpVersion) => {
+	if (dxpVersion) {
+		return WEB_CONTENTS_BY_LIFERAY_VERSION[dxpVersion];
+	}
+
+	return WEB_CONTENTS_BY_LIFERAY_VERSION['7.4'];
+};
+
 export function getWebContents(slaCurrentVersionAndProducts) {
 	const [slaCurrent, dxpVersion, ...products] = slaCurrentVersionAndProducts;
 	const webContents = [];
+
 	if (
-		products.includes(PRODUCTS.dxp) ||
-		products.includes(PRODUCTS.portal) ||
-		products.includes(PRODUCTS.commerce) ||
-		(!products.includes(PRODUCTS.partnership) &&
-			!products.includes(PRODUCTS.dxp_cloud))
+		products.some(
+			(product) =>
+				product === PRODUCTS.dxp ||
+				product === PRODUCTS.portal ||
+				product === PRODUCTS.commerce
```

Comment:
Maybe create a array with allowed products and use the `allowed.includes(product)` should be the better approach

Reply:
I agree

Reply:
but not only allowed, because could be anything.

allowedProducts, at least